packages/app/src/specs/DirectoryItem.vue

```diff
@@ -14,6 +14,7 @@
       class="text-gray-400 group-focus:text-indigo-300"
       highlight-classes="font-bold text-white"
     />
+    <slot name="run-all-specs" />
```

Comment:
I made this more clear by adding a name to the slot. In general, I think we should avoid default slots, unless it's really obvious what it is.

https://github.com/cypress-io/cypress/pull/24738/commits/eb8bc3d6fc3dcd0c0d135eed2946f2f657a0966c